components/main_page/commons/main_page_layout.lua

```diff
@@ -68,11 +68,12 @@ function MainPageLayout._makeCells(cells)
 		for _, item in ipairs(column.children) do
 			local content = {}
 			if item.content then
+				local contentBody = item.content.body
 				if item.content.noPanel then
-					table.insert(content, frame:preprocess(item.content.body))
+					table.insert(content, type(contentBody) == 'string' and frame:preprocess(contentBody) or tostring(contentBody))
```

Comment:
is it possible to get a nil return here?
if so tostring would make it to `'nil'`